test/linter.test.js

```diff
@@ -66,9 +66,11 @@ describe('Linter', () => {
                 getLinterErrors(runLinter('something', input, args.key)).should.be.deepEqual(expectedErrors);
             }
 
-            const lintAndExpectValid = async (rule, input, args = {}) => {
```

Comment:
No need to have this function be async, and it was making failures show up as unhandled promise warnings (false green on test suite)